web/i18n/helper.go

```diff
@@ -39,10 +44,44 @@ func New(r repo.Interface) (*Helper, error) {
 			return nil
 		}
 
-		_, err = bundle.LoadMessageFile(path)
+		mfb, err := ioutil.ReadAll(rs)
+		if err != nil {
+			return err
+		}
+		_, err = bundle.ParseMessageFileBytes(mfb, path)
 		if err != nil {
 			return fmt.Errorf("i18n: failed to parse file %s: %w", path, err)
 		}
+		fmt.Println("loaded", path)
+		return nil
+	}
+
+	// walk the embedded defaults
+	err := vfsutil.WalkFiles(Defaults, "/", walkFn)
```

Comment:
Just trying to understand: where does `Defaults` come from?

Reply:
in `-tags dev` builds it's from here:

https://github.com/ssb-ngi-pointer/go-ssb-room/pull/16/files#diff-6eb611b44b73353aabbe7c47960117cf3e6dd36dfc697af131048df497eb7b44R23

otherwise it's from the defaults_vfsdata.go (updated with `go generate`)